mirrors.d/almalinux.linuxpatch.com.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: almalinux.linuxpatch.com
+address:
+  http: http://almalinux.linuxpatch.com/
+  https: https://almalinux.linuxpatch.com/
+geolocation:
+  continent: Europe
+  country: UK
+  state_province: Greater London
+  city: London
+update_frequency: 12h
```

Comment:
The sync frequency should be 3 hours or less.  If it is greater than 3h the mirror will constantly show as "expired" and will not be served.

Can you update this?

Reply:
Changes applied.